packages/scenario-harness/ui/app.js

```
@@ -554,6 +612,34 @@ function wire() {
+    if (!btn) return;
+    // Capture the resting label ONCE at wireup. If we re-read it per-click,
+    // a click during the 1500 ms "doneLabel" window would capture that
+    // success label as the new "resting" text and the button stays stuck.
+    const original = btn.textContent;
+    btn.addEventListener("click", async () => {
+      btn.textContent = busyLabel;
+      btn.disabled = true;
+      try {
+        await post(endpoint, {});
+        btn.textContent = doneLabel;
+        setTimeout(() => {
```

ESLint

[error] 'setTimeout' is not defined.

(no-undef)